test_results/client.go

```diff
@@ -1,4 +1,4 @@
-package testresults
+package test_results
```

Comment:
Is there a reason this was changed? It's more conventional in Go to avoid using underscores/camelcase for package names.

Reply:
ok I will change `system_metrics` to `systemmetrics` then, the inconsistency was bothering me.

Reply:
Same, but now harmony has been restored.